packages/spor-theme-react/src/utils/focus-utils.ts

```diff
@@ -0,0 +1,16 @@
+import { CSSObject } from "@chakra-ui/react";
+
+type FocusArgs = {
+  /** The styles applied for when the element is not focused */
+  notFocus: CSSObject;
+  /** The styles applied for when the element is focused */
+  focus: CSSObject;
+};
+/**
+ * A helper function for applying focus styles in such a way that they work with Safari – which doesn't support :focus-visible.
+ */
+export const focusVisible = ({ notFocus, focus }: FocusArgs) => ({
+  _focus: focus,
+  _focusVisible: focus,
+  "&:focus:not(:focus-visible)": notFocus,
+});
```

Comment:
Dette er et fint lite utility man kan bruke for å forenkle "tastaturnavigasjons-fokus" stiler. Kommer en ny PR hvor denne tas i bruk på tvers av resten av koden vår.